test/browser/step_definitions/relying-party.js

```diff
@@ -18,3 +24,25 @@ Then("they should be redirected as an error", function () {
 
   expect(rpPage.hasErrorQueryParams()).to.be.true;
 });
+
+Then(/^they should be redirected as a success$/, function () {
+  const rpPage = new RelyingPartyPage(this.page);
+
+  expect(rpPage.isRelyingPartyServer()).to.be.true;
+
+  expect(rpPage.hasSuccessQueryParams()).to.be.true;
+});
+
+Then(/^the error should be (.*)$/, function (error_code) {
+  const rpPage = new RelyingPartyPage(this.page);
+
+  expect(rpPage.isRelyingPartyServer()).to.be.true;
+
+  expect(rpPage.isErrorCode(error_code)).to.be.true;
+});
+
+When(/^they return to a previous page$/, async function () {
+  const rpPage = new RelyingPartyPage(this.page);
+
+  await rpPage.page.goBack();
+});
```

Comment:
We had two RP files - merged them into one and kept the same naming convention as the pages.